v2/component/launcher.go

```diff
@@ -165,10 +165,14 @@ func (o *LauncherOptions) validate() error {
 }
 
 const outputMetadataFilepath = "/tmp/kfp_outputs/output_metadata.json"
+const defaultPipelineRoot = "minio://mlpipeline/v2/artifacts"
```

Comment:
Parameters and output_metadata.json are also stored under pipeline_root? Suggest making the default "minio://mlpipeline/v2/default_pipeline_root".

Reply:
I thought we only store artifacts in pipeline_root @Bobgy  can you help confirm?. In v1, we name it as "minio://mlpipeline/artifacts"

Reply:
Yes, parameters are only in MLMD, metadata are in MLMD and v1 artifact location (because we pass by argo).

Reply:
Does v2 writes output_metadata.json to pipeline_root?

Reply:
No, v2 will only write to MLMD per current design.